pandas/tests/indexing/test_indexing.py

```diff
@@ -890,6 +890,13 @@ def test_no_reference_cycle(self):
         del df
         assert wr() is None
 
+    def test_zero_index_iloc_raises(self):
+        df = pd.DataFrame([[1, 2], [3, 4]], columns=['a', 'b'])
```

Comment:
Reference issue as a comment below the function `def`.

Reply:
Can do.